src/components/common/LoginPages/sections/login-form.tsx

```diff
@@ -0,0 +1,130 @@
+import { useTranslation } from "react-i18next";
+import { useSyncLanguage } from "@/hooks/useSyncLanguage";
+import { Heading } from "@/components/common/LoginPages/ui/heading";
+import { MyInput } from "@/components/design-system/input";
+import { Link } from "@tanstack/react-router";
+import { loginSchema } from "@/schemas/login";
+import { z } from "zod";
+import { useState } from "react";
+import { SplashScreen } from "@/components/common/LoginPages/layout/splash-container";
+import { LoginFormState } from "../types";
+import { loginUser } from "@/components/common/LoginPages/hooks/dummy/login/login-button";
+import { useMutation } from "@tanstack/react-query";
+import { ErrorDialog } from "../ui/error-dialog";
+import { MyButton } from "@/components/design-system/button";
+
+export function LoginForm() {
+    useSyncLanguage();

```

Comment:
Can you include this at global level in our project such that we don't need to include again and again.